components/jenkins/init.tf

```diff
@@ -26,4 +26,10 @@ provider "azurerm" {
   skip_provider_registration = true
   alias                      = "cosmosdb"
   subscription_id            = var.cosmos_subscription_id
+}
+
+provider "azurerm" {
+  subscription_id = "8999dec3-0104-4a27-94ee-6588559729d1"
```

Comment:
Sanity check: Are we planning on re-using the same subscription id's between sds and cft?